manifests/c/CodecGuide/K-LiteCodecPack/Basic/17.3.0/CodecGuide.K-LiteCodecPack.Basic.installer.yaml

```
@@ -0,0 +1,27 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: CodecGuide.K-LiteCodecPack.Basic
+PackageVersion: 17.3.0
+Platform:
+- Windows.Desktop
+MinimumOSVersion: 10.0.0.0
+Scope: machine
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+UpgradeBehavior: install
+Installers:
+- Architecture: x86
+  InstallerType: inno
+  InstallerUrl: https://files3.codecguide.com/K-Lite_Codec_Pack_1730_Basic.exe
+  InstallerSha256: D6672DD124BCAD0873117EF5CDEE3B6891099E2C6D16AFFC7FC5149D185CC190
+  AppsAndFeaturesEntries:
+  - DisplayName: K-Lite Codec Pack 17.2.5 Basic

```

```suggestion
  - DisplayName: K-Lite Codec Pack 17.3.0 Basic
```
This must match Control Panel